File: BRACU ACM Student Chapter/Online Workshop on Programming Skills DAY 01 ( JAVA ) BRACU ACM Student chapter/K.java
import java.util.Scanner;
import java.lang.Math;
public class K{
  public static void main (String[] args){
    Scanner input = new Scanner(System.in);
    int testcase = input.nextInt();
    for(int i=1; i<=testcase; i++)
    {        
      double r;
      r = input.nextDouble();
      System.out.printf("Case %d: %.2f\n", i, (2*r*2*r)-(2*Math.acos(0.0)*Math.pow(r,2)));
    }
  }
}

File: BRACU ACM Student Chapter/Online Workshop on Programming Skills DAY 01 ( C++ ) BRACU ACM Student chapter/K.c
#include<stdio.h>
#include<math.h>
int main()
{
    int i, testcase;
    scanf("%d", &testcase);
    for(int i=1; i<=testcase; i++)
    {
        double r;
        scanf("%lf", &r);
        printf("Case %d: %.2lf\n", i, (2*r*2*r)-(2*acos(0.0)*pow(r,2)));
    }
    return 0;
}
